pkg/controller/vsphere/reconciler.go

```diff
@@ -70,6 +70,11 @@ func (r *Reconciler) create() error {
 		klog.Infof("%v: cloning", r.machine.GetName())
 		task, err := clone(r.machineScope)
 		if err != nil {
+			metrics.RegisterFailedInstanceCreate(&metrics.MachineLabels{
+				Name:      r.machine.Name,
+				Namespace: r.machine.Namespace,
+				Reason:    fmt.Sprintf("Clone task finished with error: %+v", err),
```

Comment:
Just use `%v` here, otherwise you'll get a lot of garbage information that we don't really want to collect. There's a few of these so make sure to update in other places too